### ISSUE TITLE
Master NMT path changes when media location changes

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#1462]
Original [issue 1463](https://code.google.com/p/moviejukebox/issues/detail?id=1463) created by Omertron on 2010-05-05T23:02:06.000Z:

I've been wanting to find a way around this problem for a _very_ long time
now...

Any time a path changes within the library.xml file, a full rebuild is
required for all those media files within that original path.
This is because all location paths are hard-coded with the full path inside
the xml file, and thus used by the HTML file.

The problem happens when an attached USB drive path is changed by the PCH
when you plug in another USB device.
So while I might have had USB_DRIVE_B-1 previously setup and working, a
reboot has caused it to now be USB_DRIVE_C-4  (ughh!)

This happened today after installing a 3.5&quot; SATA drive into the C200 drive 
bay.

Ideally I would have loved to simply go to the jukebox folder on the NMT
and inside the relevant skin, have a single file (locations.xml)
which I could change to the new path, and the HTML files simply use that
path as the base of all further location requests.

I understand that the skins could not currently handle that, but hopefully
that would change over time.

On the PC where I generate YAMJ output, simply changing the library.xml
file's NMT paths, would be enough to trigger
a rebuild of only the necessary HTML/jsp files.

Bottom line, I want to avoid having _all_ the XML files needing to be
rebuilt, and then copied to the NMT (even if not used).
